docs/site/DataSource-generator.md

```diff
@@ -44,6 +44,17 @@ The tool will prompt you for:
 - **Connector configuration.** If the connector is not a custom connector, the
   CLI will prompt for the connector configuration information.
 
+In case of "In-memory db" connector, you will be prompted for these additional
```

Comment:
since this is applicable for LB3 as well, and the LB3 docs has a memory connector page, would it make more sense to put it in https://loopback.io/doc/en/lb3/Memory-connector.html?

Reply:
The LB3 doc has the info already.

The behavior of the memory connector in `lb4 connector` and `lb connector` are different, so I guess this PR should suffice. Differences:

1. In LB4, the database directory will be created by the generator. In LB3, the running app will throw an error.
2. In LB4, if no name is specified, the generator will generate one based on the name of the connector and create the file in the application directory. In LB3, the behavior is "unknown" and undocumented.